dev_guide/templates.adoc

```diff
@@ -122,8 +121,9 @@ uploading the template by piping the process output to the `create` command:
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v
-ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json \
+    -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin \
+    | osc create -f -
```

Comment:
@adellape 
So horizontal scroll is preferable?  Personally, i *really* dislike having to do it when reading documentation.  I think if backslashes can preserve correctness on cut and paste (as they do for shell commands), then why not use them?
Is there a guideline doc somewhere?  I'm sure this is not the first time the issue has surfaced...